doc/getting-started/training.md

```diff
@@ -0,0 +1 @@
+../train/training.md
```

Comment:
Too much advanced topic in this training.md. 

Guide the user with a simple example and explain the out put in the `lcurve.out`

Reply:
Going to update tomorrow. It's too late in my time zone.